app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/title/TitleViewModel.kt

```diff
@@ -161,60 +161,60 @@ class TitleViewModel @Inject constructor(
-                        .flatMapCompletable { addAllResponse ->
-                            tumeKyouenRepository.updateSyncClearData(addAllResponse.data)
-                        }
-                        .observeOn(AndroidSchedulers.mainThread())
-                        .subscribe(
-                                {
-                                    mutableConnectStatus.value = ConnectStatus.CONNECTED
-                                    refresh()
-                                },
-                                {
-                                    mutableConnectStatus.value = ConnectStatus.CONNECTED
-                                    _alertMessage.value = Event(R.string.alert_error_sync)
-                                }
-                        )
+            tumeKyouenRepository.selectAllClearStage()
+                .subscribeOn(Schedulers.io())
+                .flatMap<AddAllResponse> { stages -> ServerUtil.addAll(tumeKyouenService, stages) }
+                .flatMapCompletable { addAllResponse ->
+                    tumeKyouenRepository.updateSyncClearData(addAllResponse.data)
+                }
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(
+                    {
+                        mutableConnectStatus.value = ConnectStatus.CONNECTED
+                        refresh()
+                    },
+                    {
+                        mutableConnectStatus.value = ConnectStatus.CONNECTED
+                        _alertMessage.value = Event(R.string.alert_error_sync)
+                    }
+                )
         )
     }
 
     fun requestStages(count: Int) {
         val taskCount = if (count == -1) Integer.MAX_VALUE else count
         disposable.add(
-                tumeKyouenRepository.selectMaxStageNo()
-                        .subscribeOn(Schedulers.io())
-                        .flatMap { maxStageNo -> insertDataTask.run(maxStageNo, taskCount) }
-                        .observeOn(AndroidSchedulers.mainThread())
-                        .subscribe(
-                                { successCount ->
-                                    _toastMessage.value = Event(StringResource(R.string.toast_get_stage, arrayOf(successCount)))
-                                    refresh()
-                                },
-                                {
-                                    _toastMessage.value = Event(StringResource(R.string.toast_no_stage))
-                                }
-                        )
+            tumeKyouenRepository.selectMaxStageNo()
+                .subscribeOn(Schedulers.io())
+                .flatMap { maxStageNo -> insertDataTask.run(maxStageNo, taskCount) }
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(
+                    { successCount ->
+                        _toastMessage.value = Event(StringResource(R.string.toast_get_stage, arrayOf(successCount)))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/kyouen-android/blob/190e05f3009232f6a77bdf0cc25a1773db9f7ee6/app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/title/TitleViewModel.kt#L193"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>